docker-cloud/builds/index.md

```diff
@@ -2,16 +2,10 @@
 description: Manage Builds and Images in Docker Cloud
 keywords:
 - builds, images, Cloud
-menu:
-  main:
-    parent: builds
-    weight: -80
 notoc: true
-title: Builds and Images Overview
+title: Build and images overview
```

Comment:
I really like sentence capitalization so "overview" doesn't look strange to me.
But you can always change the sentence to "Overview of build and images"